.github/workflows/build-standalone.yml

```
@@ -170,12 +238,155 @@ jobs:
+          pc init
+          pc list
+
+      # "snap/snapcraft.yaml" points PC_INTERNAL_STATE_DIR at the per-user
+      # directory snapd gives the snap, so that PartCAD's cache, sandboxes and
+      # git clones do not land in ~/.partcad. Nothing about the commands above
+      # succeeding would show that, so check it from both ends: that snapd
+      # exports the variable, and that PartCAD acted on it. Note that snapd
+      # creates ~/snap/partcad/common for every snap it runs, so its mere
+      # existence proves nothing and is not what is asserted.
+      - name: Check the snap keeps its state in the snap directory
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:4:19: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)